### PR TITLE
L 452 navbar nav

### PR DESCRIPTION
**Pull request by vsevolod-adrianov**
clear "navbar - nav" tests

### CODE REVIEW

**Comment by vsevolod-adrianov** [comment]
Clear Navbar - Nav & Navbar - Container

**Comment by RomE-HK** [comment]
Pardon me, I don't get it. Are this PR for NavBar Navs or NavBar Container task?

File: jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/navbar/NavbarContainer.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package io.github.com.sections.navbar;
+
+import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+
+public class NavbarContainer extends Section {
+    @UI("#navbar-containers-centred") public NavbarSimpleLinks navLinks1;
+    @UI("#navbar-containers-expanded") public NavbarComplexLinks navbarComplexLinks;
```
Comment (RomE-HK) [inline jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/navbar/NavbarContainer.java]: Are there really the same elements as in NavbarNav.java so we could use same elements' types?

File: jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java [hunk 2]
```diff
@@ -113,16 +112,7 @@
...
-import io.github.com.sections.navbar.NavbarExternalContent;
-import io.github.com.sections.navbar.NavbarForm;
-import io.github.com.sections.navbar.NavbarPlacement;
-import io.github.com.sections.navbar.NavbarSection;
-import io.github.com.sections.navbar.NavbarSupportedContent;
-import io.github.com.sections.navbar.NavbarText;
-import io.github.com.sections.navbar.NavbarWithDropdown;
-import io.github.com.sections.navbar.NavbarWithInputGroupForm;
-import io.github.com.sections.navbar.ScrollSpyNav;
+import io.github.com.sections.navbar.*;
```
Comment (RomE-HK) [inline jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java]: We shouldn't use wildcards imports

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/complex/Navbar.java [hunk 3]
```diff
@@ -1,18 +0,0 @@
...
-
-public class Navbar extends Section {
-
-    @UI("//*[contains(@class, 'nav-item')]") public WebList navbarLinks;
-    @UI(".navbar-brand") public Link navbarBrand;
-    @JDropdown(expand = ".navbar-toggler-icon", value = ".navbar-nav", list = "a")
-    public Collapse collapseButton;
-    @UI("//*[contains(@class, 'nav-link dropdown-toggle')]")
-    public Dropdown navbarDropdown;
-}
```
Comment (RomE-HK) [inline jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/complex/Navbar.java]: Why the whole class was deleted?

**Comment by RomE-HK** [comment]
Changes requested
And, please, pay attention to Codacy checks

File: jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java (hunk 2 above)
Comment (elv1s42) [inline jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java]: Codacy Issue found: [Avoid unused imports such as 'io.github.com.sections.navbar'](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4457828)

**Comment by elv1s42** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/navbar/NavbarComplexLinks.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4457828&bid=15064762)

**Comment by elv1s42** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/navbar/NavbarComplexLinks.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4457828&bid=15064762)

File: jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java [hunk 4]
```diff
@@ -548,7 +553,7 @@
     @UI("#nav-pills-drop")
     public static NavsPillsWithDropdown navsPillsWithDropdown;
     @UI("#navbar-dark-colorscheme")
-    public static NavbarColorScheme navbarDarkColorScheme;
+    public static io.github.com.sections.navbar.NavbarColorScheme navbarDarkColorScheme;
```
Comment (elv1s42) [inline jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java]: Codacy Issue found: [Unnecessary use of fully qualified name 'io.github.com.sections.navbar.NavbarColorScheme' due to existing import 'io.github.com.sections.navbar.NavbarColorScheme'](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4457828)

**Comment by elv1s42** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/navbar/NavbarComplexLinks.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4457828&bid=15064762)

File: jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java (hunk 4 above)
Comment (elv1s42) [inline jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java]: Codacy Issue found: [Unnecessary use of fully qualified name 'io.github.com.sections.navbar.NavbarColorScheme' due to existing import 'io.github.com.sections.navbar.NavbarColorScheme'](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4457828)

**Comment by elv1s42** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/navbar/NavbarComplexLinks.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4457828&bid=15064762)

**Pull request merged**